helpdesk_timesheet/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+#. Assign a project and task to the ticket
+#. Register the time you spent on it
```

Comment:
Add a small image for this?

Reply:
what kind of pic do you suggest?

Reply:
Something like [helpdesk_mgmt](https://github.com/OCA/helpdesk/tree/11.0/helpdesk_mgmt) readme. A screenshot where the time can be registered (of course it will be in the ticket form, but it's nice to see it before installing the module)

Reply:
okay i can add this !